tests/mentorship_relation/test_api_cancel_relation.py

```diff
@@ -27,8 +28,8 @@ def setUp(self):
         # create new mentorship relation
 
         self.mentorship_relation = MentorshipRelationModel(
-            action_user_id=self.first_user.id,
-            mentor_user=self.first_user,
+            action_user_id=self.admin_user.id,
```

Comment:
Instead of making changes in the existing code, try adding another block within the uninvolved test case itself to solve the problem since this change will affect other test cases as well

Reply:
Done!